packages/storybook-react/config/preview.js

```diff
@@ -0,0 +1,19 @@
+import '@utrecht/component-library-vue/dist/style.css';
```

Comment:
Is this `component-library-vue` import correct, or should it be `component-library-css`? In any case the correct dependency should also be added to `package.json`.

Reply:
This `component-library-css` should be the correct one